CHANGES.txt

```diff
@@ -1,3 +1,32 @@
+0.3
+---
+
+- Python 3 is now supported! This is possible thanks to python-ldap supporting
+  python 3
+
+- BREAKING CHANGE: pyramid_ldap now depends on python-ldap >= 3.0, which
+  supports python 2 and 3. If you upgrade pyramid_ldap without upgrading
+  python-ldap, you will see failures where python-ldap receives text arguments
+  instead of bytes. See below for more information.
+
+- python-ldap >= 3.0 stopped accepting `bytes` type arguments for many
+  `LDAPObject` methods. We now use the `pyramid.compat.text_` function to
+  support text arguments for python 2 and 3.
+
+- The  method typically deals with
+  untrusted `login` and `password` inputs (i.e. user inputs).
+
+- In versions 0.2 and lower, invalid LDAP strings passed to
+  `pyramid_ldap.Connector.authenticate` would raise `ldap.FILTER_ERROR`. In 0.3
+  and up we use `ldap.filter.escape_filter_chars` to properly escape these prior
+  to running the LDAP search. Applications using pyramid_ldap have always been
+  responsible for making sure the `login` value fits the `ldap.login_filter_tpl`
+  setting in their application. However, if you relied on `ldap.FILTER_ERROR` to
+  catch bad username formats (such as `CORP\username`, where the unescaped `\`
+  is disallowed in LDAP searches), note that now `authenticate` will return
+  `None` instead.
+
```

Comment:
For reStructuredText syntax and in Sphinx, single backticks render as italic, double backticks render as inline code.
![google chrome005](https://user-images.githubusercontent.com/102112/39497500-6de8a47a-4d58-11e8-94ab-72a2914a3875.png)

Markdown uses single backticks for inline code. I'm not sure which you intended, but it looks like you attempted inline code. If so, then please change backticks from single to double throughout, e.g., ``ldap.FILTER_ERROR``. Coincidentally, that will also fix the problem where a backslash escapes a backtick and the inline code formatting does not terminate until the next backtick. See http://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#inline-markup